solr/server/etc/security.policy

```diff
@@ -219,6 +219,9 @@ grant {
   permission java.io.FilePermission "${solr.allowPaths}", "read,write,delete,readlink";
   permission java.io.FilePermission "${solr.allowPaths}${/}-", "read,write,delete,readlink";
 
+  permission java.io.FilePermission "${solr.sharedLib}", "read,readlink";
```

Comment:
Note that `solr.sharedLib` may contain a comma separated list of paths, which may not be supported in policy files. Is there a way to define policy programmatically?

Reply:
True, it would fail with multiple paths (Note that the `allowPaths` just above is the same though). Another thing I don't really like is that these system properties are just "defaults", since the values are really just to populate `solr.xml`.
May be good enough though? Maybe the solution is to make it easy/documented how/when to update the policy for advanced usecases. Last I checked it wasn't easy to use a different policy (or amend one).
On the other hand, I don't know how much effort does the security manager configuration deserves, given it's deprecated.

Reply:
Yea, it will go away, so agree on that. One option could be to write a SolrCLI tool to generate/update the policy file. That way, `bin/solr` could run a tool that generates correct policy file based on whatever env/prop active and then start the main Solr process.

Reply:
Furthermore, in a containerized world, thus with lots of read-only parts of the file system except for "/var", the point of enforcing file system checks in production via a security manager feels overly redundant -- too much of a pain to maintain.